megatron/tokenizer/gpt2_tokenization.py

```diff
@@ -174,6 +176,7 @@ def __init__(self, vocab_file, merges_file, errors='replace',
         self.special_tokens = {}
         self.special_tokens_decoder = {}
         self.set_special_tokens(special_tokens)
+        self.max_token_len_cache=max_token_len_cache
```

Comment:
```suggestion
        self.max_token_len_cache = max_token_len_cache
```

I guess we need to write a style guide so the code is consisten. `a=b` is typically used inside function args, but `a = b` otherwise. At least in HF style.